### ISSUE TITLE
Use Splash as a proxy

### ISSUE THREAD

**Issue by lolobosse** [scrapinghub/splash#1080]
Hello there, 

I'm wondering if Splash could have **transparent** endpoint:
It just does the request and returns the response transparently (no metadata, **no rendering**, nothing).

My use case is the following:
We use a splash service which is configured with all the rotating proxies (via splash proxyprofiles) and the security linked to them (IP policy and so on). Until now, we were only parsing some webpages so it worked great. However, from now on, we try to query REST APIs and then parse the links that they give us back. I still want to execute the API queries via Splash (because of the rotating proxies) but `render.html` or other endpoints are adding a complexity that I need to remove once the request returns.

Basically I'm wondering if that's possible to use spash as a simple proxy but with the same ease use as before with `SplashRequest`

**Comment by Gallaecio** [comment]
The usual approach in such an scenario is to fetch the relevant cookies from the first Splash request, and perform regular Scrapy requests that include those cookies, which usually does the job. Although you may need to refresh those cookies now and then with additional Splash requests.
